packages/perseus-editor/src/util/deprecated-widgets/modernize-widgets-utils.ts

```diff
@@ -14,17 +14,12 @@ const widgetRegExes = [/input-number \d+/]; // We can add more regexes here in t
 export const convertDeprecatedWidgets = (
     json: PerseusRenderer,
 ): PerseusRenderer => {
-    // If there's no widgets that require conversion, return the original json
-    if (!conversionRequired(json)) {
-        return json;
-    }
-
```

Comment:
Don't need to check here any more, as we're doing it directly in the editors. This allows us to know when the json has been updated! :)